lighthouse-core/config/config.js

```diff
@@ -331,6 +320,101 @@ class Config {
   }
 
   /**
+  * Filter out any unrequested aggregations from the config. If any audits are
+  * no longer needed by any remaining aggregations, filter out those as well.
+  * @param {!Object} config Lighthouse config object.
+  * @param {!Array<string>} aggregationNames Name values of aggregations to include.
+  * @return {undefined} config is mutated, instead of returning
+  */
+  static generateConfigOfAggregations(config, aggregationNames) {
```

Comment:
"generate" implies something new is being created. Might want to use something like `removeUnrequestedAggregations`?